Ginger/GingerCore/ALM/OctaneCore.cs

```diff
@@ -118,20 +262,7 @@ public override ObservableList<ExternalItemFieldBase> GetALMItemFields(Backgroun
         {
             ObservableList<ExternalItemFieldBase> fields = new ObservableList<ExternalItemFieldBase>();
             string resourse = string.Empty;
-            AlmResponseWithData<AlmDomainColl> domains = Task.Run(() =>
-            {
-                return octaneRepository.GetLoginProjects(ALMCore.DefaultAlmConfig.ALMUserName, ALMCore.DefaultAlmConfig.ALMPassword, ALMCore.DefaultAlmConfig.ALMServerURL);
-            }).Result;
-            AlmDomain domain = domains.DataResult.Where(f => f.DomainName.Equals(ALMCore.DefaultAlmConfig.ALMDomain)).FirstOrDefault();
-            ProjectArea project = domain.Projects.Where(p => p.ProjectName.Equals(ALMCore.DefaultAlmConfig.ALMProjectName)).FirstOrDefault();
-            LoginDTO loginDto = new LoginDTO()
-            {
-                User = ALMCore.DefaultAlmConfig.ALMUserName,
-                Password = ALMCore.DefaultAlmConfig.ALMPassword,
-                Server = ALMCore.DefaultAlmConfig.ALMServerURL,
-                SharedSpaceId = domain.DomainId,
-                WorkSpaceId = project.ProjectId
-            };
+            LoginDTO loginDto = GetLoginDTO();
```

Comment:
Codacy found an issue: [Rename 'loginDto' which hides the field with the same name.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6317725)